integration_tests/sdk/aqueduct_tests/naming_test.py

```diff
@@ -113,22 +113,7 @@ def test_naming_collision_with_different_types(client, data_integration):
     # An overwrite is invalid because the operators are of different types.
     table_artifact = extract(data_integration, DataObject.SENTIMENT, op_name="sql query")
 
-    # Function collides with existing sql artifact
-    _ = extract(data_integration, DataObject.SENTIMENT, op_name="dummy_model")
-    with pytest.raises(InvalidUserActionException):
-        dummy_model(table_artifact)
-
     # SQL collides with existing function
     _ = dummy_sentiment_model(table_artifact)
     with pytest.raises(InvalidUserActionException):
         _ = extract(data_integration, DataObject.SENTIMENT, op_name="dummy_sentiment_model")
-
-
-def test_naming_collision_with_dependency(client, data_integration):
```

Comment:
Removing this because the following is now allowed:

```
wines = demodb.sql("select * from wine;")

@op
def f1(df):
    return df

@op
def f2(df):
    return df

a = f1(wines)
b = f2(a)
c = f1(b)
```

Reply:
Can you also add this as a test case and validate the artifact names?